lib/instrumentation.js

```diff
@@ -124,20 +124,28 @@ function wrapQueriable(shim, queriable, isPoolQuery) {
     return false
   }
 
-  var proto = Object.getPrototypeOf(queriable)
+  const proto = Object.getPrototypeOf(queriable)
+
+  let describe
   if (isPoolQuery) {
-    shim.recordQuery(proto, 'query', describePoolQuery)
+    describe = describePoolQuery
   } else {
-    shim.recordQuery(proto, 'query', describeQuery)
+    describe = describeQuery
     shim.setInternalProperty(proto, '__NR_databaseName', null)
   }
 
+  shim.recordQuery(proto, 'query', describe)
+
+  if (queriable.execute) {
```

Comment:
this was the bit that addressed the ticket. the rest was fixing tests and auto linting fixes